Restructure folders

arauschert opened the pull request:
- now all files only reference files in subdirectories and not parallel directories
- clarified use of logical and concrete scenarios

Closes #65

arauschert committed:
Restructure folders

- now all files only reference files in subdirectories and not parallel
directories
- clarified use of logical and concrete scenarios

jdsika commented:
I personally do not like folders with capitals as it seems annoying in Linux systems.

Vorschlag:

> Folders and filename are lowercase and words are separated by - (except the _ as separator for the suffix).

jdsika commented:
Remove "." inside file names

arauschert committed:
Address review comments

- remove . in middle of scenario filenames
- change folder and filenames to snake case

jdsika commented on README.md:
```diff
@@ -31,18 +31,18 @@ The focus of the here provided scenarios is on securing the planning aspects of
 The execution of the concrete scenarios in the open source tools "esmini", a basic OpenSCENARIO player, and "openPASS", a simulation platform for traffic simulation, is described on Windows:
 
-_Note:_ Currently only esmini supports the OpenSCENARIO 1.1 format. For openPASS please use the [release v0.3.2](https://github.com/asam-oss/OSC-ALKS-scenarios/releases/tag/v0.3.2) with scenarios in OpenSCENARIO 1.0 format.
+_Note:_ Currently only esmini supports the OpenSCENARIO 1.3 format. For openPASS please use the [release v0.3.2](https://github.com/asam-oss/OSC-ALKS-scenarios/releases/tag/v0.3.2) with scenarios in OpenSCENARIO 1.0 format.
```
Is it still the right tag for openPASS?

arauschert replied:
There was no test with a newer version of openPASS, so this sentence is still valid.
